### DISCUSSION TITLE
[BUG] GitHub profile cannot be updated correctly if we enter it from the following/followers tab.

### DISCUSSION THREAD

**Discussion by higuoxing** [community/community#37516] [Other Feature Feedback, Questions, & Ideas] [answered]
The profile on the left side cannot be updated correctly if we enter it from someone's following/followers tab.
I have attached a video to show how to reproduce the bug.

https://user-images.githubusercontent.com/21099318/198305142-254ee61b-4e56-45f9-b64a-2cae4c512b60.MOV

**Comment by gonzalo-cordova-pou** [comment]
Same!!

**Comment by bananatron** [comment]
The (probably turbo) response is coming back w/ only profile partial w/out knowing it needs to update the left pane 🙊

**Comment by radar-blip** [comment, marked as answer]
Same happening here. If you refresh the page it should go back to normal.
Seems like a temporary thing, Hopefully it will be fixed soon. 

Update: Seems this is fixed now.
